donations.yaml

```diff
@@ -46,4 +48,17 @@ cisco:
     - $10000 to Kevin McCarthy, who doesn't think climate change is real, and managed to get a 0% rating from Planned Parenthood and 100% from the <abbr title="National Rifle Association">NRA</abbr>.
-    - $5000 to Kevin Brady, who voted against making sexual orientation a protected class and <a href="https://www.aclu.org/other/promoting-access-contraception-and-opposing-threats-its-availability-home-and-abroad">introduced a bill</a> to prevent minors from getting prescription contraceptives.
\ No newline at end of file
+    - $5000 to Kevin Brady, who voted against making sexual orientation a protected class and <a href="https://www.aclu.org/other/promoting-access-contraception-and-opposing-threats-its-availability-home-and-abroad">introduced a bill</a> to prevent minors from getting prescription contraceptives.
+
+ebay:
+  url: https://www.fec.gov/data/receipts/?two_year_transaction_period=2018&data_type=processed&contributor_name=C00342394&min_date=01%2F01%2F2017&max_date=04%2F09%2F2018
+  donations:
+    - $2500 to Steve Scalise, who has one of "the most <a href="https://www.hrc.org/resources/congressional-scorecard">anti-LGBT</a> reputations of any lawmaker" and who <a href="https://www.nytimes.com/2015/01/01/us/politics/much-of-david-dukes-91-campaign-is-now-in-louisiana-mainstream.html?hp&action=click&pgtype=Homepage&module=first-column-region&region=top-news&WT.nav=top-news">said</a> he was "like David Duke without the baggage".
+    - $10000 to Kevin McCarthy, who doesn't think climate change is real, and managed to get a 0% rating from Planned Parenthood and 100% from the <abbr title="National Rifle Association">NRA</abbr>.
+    - $9200 to Orrin Hatch, who was one of 22 Senators to sign a letter encouraging the US to withdraw from the Paris agreement, has an A+ rating from the <abbr title="National Rifle Association">NRA</abbr>, and was <a href="https://www.washingtonpost.com/opinions/its-time-to-undo-the-federal-land-grab-of-bears-ears/2017/04/25/1b40ba40-2910-11e7-b605-33413c691853_story.html?utm_term=.b538e38811b7">key in supporting</a> the drastic reduction of Bears Ears National Monument in Utah.
+
+dell:
+  url: https://www.fec.gov/data/receipts/?two_year_transaction_period=2018&data_type=processed&contributor_name=C00369751&min_date=01%2F01%2F2017&max_date=04%2F07%2F2018
+  donations:
+    - $50000 to Paul Ryan. Yes, that Paul Ryan, the one who makes his staffers read <i>Atlas Shrugged</i>.
+    - $55000 to the <abbr title="National Republican Congressional Committee">NRCC</abbr>, which works to elect Republicans to the United States House of Representatives.
```

Comment:
I have been leaving off the Congressional committee donations, because:
  - every company seems to do it
  - it seems less obviously bad than supporting specific politicians
Open to argument here though.

Reply:
That makes sense! Good to find some balance of what to include. There’s plenty to pick from at Dell! I found a few more for them but wanted to wait on this PR first. Thanks for fixing this up!